src/Microsoft.PowerShell.Security/security/CredentialCommands.cs

```diff
@@ -79,6 +79,19 @@ public string Title
 
         private string _title = UtilsStrings.PromptForCredential_DefaultCaption;
 
+        /// <summary>
+        /// Gets and sets the Re-enter Password on the window promt.
+        /// </summary>
+        [Parameter(Mandatory = false, ParameterSetName = messageSet)]
+        public SwitchParameter ReEnterPassword
+        {
+            get { return _reEnterPassword; }
+
+            set { _reEnterPassword = value; }
+        }
+
+        private bool _reEnterPassword;
```

Comment:
Please use auto-implemented property:
```suggestion
        public SwitchParameter ReEnterPassword { get; set; }
```